lib/file.js

```diff
@@ -119,6 +126,26 @@ client.uploadFile = function(userId, fileURI, options) {
   }
 };
 
+client.processQueue = function() {
```

Comment:
Where is this function called from? It it called from the application consuming the module?